apps/api/v1/test/lib/bookings/_patch.integration-test.ts

```diff
@@ -0,0 +1,88 @@
+import type { Request, Response } from "express";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { createMocks } from "node-mocks-http";
+import { describe, it, expect } from "vitest";
+
+import prisma from "@calcom/prisma";
+
+import handler from "../../../pages/api/bookings/[id]/_patch";
+
+type CustomNextApiRequest = NextApiRequest & Request;
+type CustomNextApiResponse = NextApiResponse & Response;
+
+describe("PATCH /api/bookings", () => {
+  it("Returns 403 when user has no permission to the booking", async () => {
+    const memberUser = await prisma.user.findFirstOrThrow({ where: { email: "member2-acme@example.com" } });
+    const proUser = await prisma.user.findFirstOrThrow({ where: { email: "pro@example.com" } });
+    const booking = await prisma.booking.findFirstOrThrow({ where: { userId: proUser.id } });
+
+    const { req, res } = createMocks<CustomNextApiRequest, CustomNextApiResponse>({
+      method: "PATCH",
+      body: {
+        title: booking.title,
+        startTime: booking.startTime.toISOString(),
```

Comment:
Patching with the same data for now because we are testing permissions. 

Will add test to test the patching functionality itself.